packages/vscode-ui5-language-assistant/test/suite/validations/flex-disabled/flex-disabled-spec.ts

```diff
@@ -0,0 +1,54 @@
+import * as vscode from "vscode";
+import { resolve, dirname } from "path";
+import { deactivate } from "../../../../src/extension";
+import { setXMLContent, testDiagnostics, sleep } from "../../test-utils";
+
+const pkgJsonPath = require.resolve(
+  "vscode-ui5-language-assistant/package.json"
+);
+
+const rootPkgFolder = dirname(pkgJsonPath);
+const testFixtureValidationsFolder = resolve(
```

Comment:
why do we need `testFixtureValidationsFolder` variable if we only use it once?
can we `inline` it into the resolve in line 19?